app/views/static_pages/home.html.erb

```diff
@@ -1,2 +1,11 @@
 <%= render "static_pages/slide" %>
-<%= render "static_pages/productCard" %>
+
+<div class="row">
+<% @books.each do |e| %>
```

Comment:
tìm hiểu render collection partial. ở đây ko cần for each mô

Reply:
oke chị đã fix ạ